docs/code-samples/getting-started/get-local-dependencies.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+# Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+set -euo pipefail
+
+# Create .lib directory
+if [[ -d .lib ]]; then
+  rm -r .lib
+fi
+mkdir .lib
+
+# Get the dependency list
```

Comment:
is this something we want to commit?

Reply:
I think it was very useful when running the tutorials locally, before the release. It is probably a script that we could all use, so I would say yes. 
I could also move it to the `../scripts` folder or somewhere else, if we think that our users would never use it, but possibly they would also find it useful in some cases (e.g. if they want to try out the tutorials against current `main` or if they have modified a library locally).

Reply:
seems like a convenient script to have for us. if we want to commit it such our users see it, we should probably provide a corresponding .bat script as well.

Reply:
can we move it to a separate PR and discuss it there? Not sure we want to ship it with the daml assistant template, customers are not even expected to clone our repo for that. Agree that this is useful in practice though

Reply:
Also, if we do breaking changes to `main`, the quickstarter in the Daml assistant would not work with `get-local-dependencies`